fixup

```diff
@@ -0,0 +1,11 @@
+cat >cjs/package.json <<!EOF
```

Comment:
Unnecessary file?

Reply:
this is necessary since esm and cjs have different import entries. You can check the package.json file.

Reply:
I believe there's a better way to achieve this, but let's keep this for now